src/conversion/model-to-view-converters.js

```diff
@@ -640,8 +639,11 @@ class HighlightAttributeElement extends ViewAttributeElement {
  * described by this object.
  *
  * Each element can handle displaying highlight separately by providing `addHighlight` and `removeHighlight` custom
- * properties. Those properties are passed `HighlightDescriptor` object upon conversion and should use it to
- * change the element.
+ * properties:
```

Comment:
In the docs above add prepositions before "highlight" and "content highlight" cause they are used as nouns.